server/itx/server.go

```diff
@@ -115,7 +115,7 @@ func (s *Server) Stop(ctx context.Context) error {
 	if err := s.p2pAgent.Stop(ctx); err != nil {
 		return errors.Wrap(err, "error when stopping P2P agent")
 	}
-	if err := s.dispatcher.Stop(ctx); err != nil {
+	if err := s.dispatcher.Stop(ctx); err != nil {// notest
```

Comment:
I don't think this line could be excluded

Reply:
yes